examples/dmd/de_dg_advection_greedy.cpp

```diff
@@ -0,0 +1,1621 @@
+                cout << "GLVis visualization disabled.\n";
+            }
+        }
+        else
+        {
+            sout << "parallel " << num_procs << " " << myid << "\n";
+            sout.precision(precision);
+            sout << "solution\n" << *pmesh << *u_gf;
+            sout << flush;
+        }
+    }
+
+    // 13. Define the time-dependent evolution operator describing the ODE
+    //     right-hand side, and perform time-integration (looping over the time
+    //     iterations, ti, with a time-step dt).
+    FE_Evolution adv(*m, *k, *B, prec_type);
+
+    StopWatch fom_timer, dmd_training_timer, dmd_prediction_timer;
+
+    fom_timer.Start();
+
+    double t = 0.0;
+    vector<double> ts;
+    adv.SetTime(t);
+    ode_solver->Init(adv);
+    CAROM::Vector* init = NULL;
+    CAROM::CSVDatabase csv_db;
+
+    fom_timer.Stop();
+
+    CAROM::DMD* dmd_U = NULL;
+
+    if(de)
+    {
+        u_gf->SetFromTrueDofs(*U);
+        init = new CAROM::Vector(U->GetData(), U->Size(), true);
+
+        ts.push_back(t);
+
+        double rel_diff = 0.;
+
+        //18. Create the parametric DMD.
+        std::fstream fin(io_dir + "/parameters.txt", std::ios_base::in);
+        double curr_param;
+        std::vector<std::string> dmd_paths;
+        std::vector<CAROM::Vector*> param_vectors;
+
+        while (fin >> curr_param)
+        {
+            double curr_f_factor = curr_param;
+
+            // INPUT / OUTPUT potential issue
+            dmd_paths.push_back(io_dir + "/" + to_string(curr_f_factor));
+            CAROM::Vector* param_vector = new CAROM::Vector(1, false);
+            param_vector->item(0) = curr_f_factor;
+            param_vectors.push_back(param_vector);
+        }
+        fin.close();
+
+        CAROM::Vector* desired_param = new CAROM::Vector(1, false);
+        desired_param->item(0) = f_factor;
+
+        dmd_training_timer.Start();
+
+        CAROM::getParametricDMD(dmd_U, param_vectors, dmd_paths, desired_param,
+                                "G", "LS", closest_rbf_val);
+
+        dmd_U->projectInitialCondition(init);
+
+
+        dmd_training_timer.Stop();
+        delete desired_param;
+
+        // Compare the DMD solution to the FOM Solution
+        if (true_solution_u == NULL)
+        {
+            ifstream solution_file;
+            ostringstream sol_name;
+            ostringstream target_name;
+            sol_name << io_dir << "/dg_advection_greedy" << to_string(
+                         target_f_factor) << "-final." << setfill('0') << setw(6) << myid;
+
+            solution_file.open(sol_name.str().c_str());
+
+            true_solution_u = new Vector(U->Size());
+            true_solution_u->Load(solution_file, U->Size());
+            solution_file.close();
+            tot_true_solution_u_norm = sqrt(InnerProduct(MPI_COMM_WORLD,
+                                            *true_solution_u, *true_solution_u));
+        }
+
+        CAROM::Vector* result_U = dmd_U->predict(t_final);
```

Comment:
The timer after running the third command still prints 0. I think the timer is missing around `result_U`:
```
        dmd_prediction_timer.Start();
        CAROM::Vector* result_U = dmd_U->predict(t_final);
        dmd_prediction_timer.Stop();
```